08_cell_line_prediction/plot_stratified_drug_regression.ipynb

```diff
@@ -0,0 +1,771 @@
+{
```

Comment:
<p>I guess I haven't seen the whole table, but is it weird that you have positive correlation in the training data and no/negative correlation in the test data? Is that just stratification noise?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yeah, I think in this case it's probably just stratification/cross-validation noise combined with overfitting, in the sense that the best model in terms of validation set performance fits the training data closely but doesn't perform as well on the test data. Ultimately predicting erlotinib sensitivity in this dataset seems to be a pretty hard problem, so getting better than random correlation is difficult for some of the cross-validation folds (although across all the folds/iterations we still end up with median correlations that are greater than 0).</p>